geopyspark/geotrellis/layer.py

```diff
@@ -1028,6 +1028,68 @@ def from_numpy_rdd(cls, layer_type, numpy_rdd, metadata, zoom_level=None):
 
         return cls(layer_type, srdd)
 
+    @classmethod
+    def from_rasterframe(cls, rasterframe, zoom_level=None):
+        """Creates a ``TiledRasterLayer from a ``pyrasterframes.RasterFrame``.
+
+        Note:
+            ``pyrasterframes`` must be installed, its associated jar
+            has to be added to the Spark path, and ``RasterFrames`` needs
```

Comment:
``RasterFrames`` needs to be enabled?  Do you mean to say that ``.withRasterFrames()`` needs to be called on the spark session object?

Reply:
Yeah, should I make that more clear in the docs?

Reply:
You could easily say "``RasterFrames`` needs to be initialized via the ``.withRasterFrames()`` extension method on the active SparkSession object."

Reply:
That's true. I'll make that change now